climada/hazard/wildfire.py

```diff
@@ -0,0 +1,1387 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define WildFire class.
+"""
+
+__all__ = ['WildFire']
+
+import logging
+import warnings
```

Comment:
Is this used? needed?

Reply:
to my understanding yes - altough it depends on code prior to my work on this module

Reply:
Where is the warnings module used? 

Please do not mark a conversation as resolved  if it is not, since this mask it for the reviewer. Thanks!

Reply:
sorry, you're right. I've remove the warnings (294ad131)